INTRODUCTION.md

```diff
@@ -1278,6 +1278,17 @@ The latest stored offset will be automatically committed every
 Broker based consumer groups (requires Apache Kafka broker >=0.9) are supported,
 see KafkaConsumer in rdkafka.h or rdkafkacpp.h
 
+##### Static Consumer groups
+
+By default Kafka consumers are rebalanced each time a new consumer joins
+the group or an existing member leaves. This is what is known as a dynamic
+membership. Apache Kafka >= 2.3.0 introduces static membership.
+Unlike dynamic membership, members can leave and rejoin a group
+within the `session.timeout.ms` without triggering a rebalance.
```

Comment:
" and without loosing their partition assignment", but in proper english, which is the net effect